src/SSW.SophieBot/SSWSophieBot/dialogs/GetExpertsByLocationDialog/language-generation/en-us/GetExpertsByLocationDialog.en-us.lg

```diff
@@ -114,26 +115,38 @@
                 "weight": "Bolder"
             }
         ]
+    },
+    {
+        "type": "Column",
+        "width": "stretch",
+        "items": [
+            {
+                "type": "TextBlock",
+                "text": "Billed",

```

Comment:
`Billed` column header should have an arrow down icon.

Reply:
We are sorting by skill level in this list